extensions/wikia/UserRenameTool/UserRenameToolTask.class.php

```diff
@@ -41,12 +35,13 @@ public function renameUser( array $wikiCityIds, array $params ) {
 		$this->staffLog(
 			'start',
 			$params,
-			\RenameUserLogFormatter::start(
+			UserRenameToolHelper::getLog(
+				'userrenametool-info-started',
 				$params['requestor_name'],
 				$params['rename_old_name'],
 				$params['rename_new_name'],
 				$params['reason'],
-				[ $this->getTaskId() ]
+				[$this->getTaskId()]
```

Comment:
Just a note, the guidelines have us include padding spaces when array brackets are used like grouping parenthesis;
```php
[ 'first', 'second', 'third' ]
```
and no space when used to index into an array:
```php
$params['foo'] = 'bar';
```

https://github.com/Wikia/guidelines/blob/master/PHP/CodingConventions.md#array-syntax